src/test-run/index.js

```diff
@@ -84,7 +84,7 @@ export default class TestRun extends AsyncEventEmitter {
 
         this.disablePageReloads = test.disablePageReloads || opts.disablePageReloads && test.disablePageReloads !== false;
 
-        this.session = SessionController.getSession(this);
+        this.session = SessionController.getSession(this, test.disablePageCaching || opts.disablePageCaching);
```

Comment:
Let's save `disablePageCaching ` in a property in the current instance and avoid adding an additional argument to `getSession`. Passing boolean flags considered as a flawed API design sometimes: https://softwareengineering.stackexchange.com/a/148058